conf/ocsci/ocp-4.3-upgrade.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Use this config for upgrading of OCP 4.3 cluster
+UPGRADE:
+  ocp_upgrade_path: "registry.svc.ci.openshift.org/ocp/release"
+  ocp_upgrade_version: ""
```

Comment:
If the intention here is to use the default value defined here:  https://github.com/red-hat-storage/ocs-ci/pull/1491/files#diff-4f80328bffaf5599157179f9e1a09464R117 
Then you should delete this line completly otherwise you will overwrite the value to empty string and you will have nothing in this value.

Reply:
its a place holder, not sure where to fetch lastest nightly image as it done for cos-operator images